getting_started.md

```diff
@@ -24,14 +24,17 @@ or
 1. In windows command prompt, run `python --version`
 2. Install Python 3.8 from the Microsoft Store
 
-## Configure the FrOST Autogen Software
-1. In WSL, clone the [Docs Repo](https://github.com/fpga-open-speech-tools/docs) into the `[FrOST Repos]` Directory 
+## Configure the Frost Autogen Software
```

Comment:
Looking good!  Since FrOST is an acronym, I feel like we should capitalize everything but the r.  What are your guys' thoughts?

Reply:
Yes, let's make the official acrynom all caps except the r, i.e. FrOST

Reply:
Updated with 03dafb7